spring-boot-project/spring-boot/src/main/java/org/springframework/boot/liquibase/CommonsLoggingLiquibaseLogger.java

```diff
@@ -32,107 +32,109 @@
  */
 public class CommonsLoggingLiquibaseLogger extends AbstractLogger {
```

Comment:
Looks like that file was touched for no good reason. Can you please revert that part?

Reply:
The functionality about rollback was introduced in liquibase-core 3.6 and at the same time logging was changed in that version. In order to compile the project after switching to that version I had to change logging implementation. You suggest that this part shouldn't be included in this PR as it will be done separately when upgrading to version 3.6. Did I get it right?

Reply:
Thanks for the feedback. Can you please submit a PR to upgrade to 3.6 only? You can then have this PR cleanly on top with only the feature that is targeted here.